examples/python/Dockerfile

```diff
@@ -5,17 +5,19 @@ ENV PYTHONBUFFERED 1
 WORKDIR /app
 RUN git clone https://github.com/twilio/twilio-python.git
 WORKDIR /app/twilio-python/
+#TODO: Change to main when GA
 RUN git fetch && git checkout 8.0.0-rc && git pull
 
-RUN rm -rf twilio/rest tests
+RUN rm -rf tests/ twilio/rest/
 
-COPY twilio/rest twilio/rest
+COPY twilio twilio
```

Comment:
Should this be copying twilio/rest? Since only twilio/rest is being removed in the previous line

Reply:
Think it works either way. Wanted to keep it consistent with [node](https://github.com/twilio/twilio-oai-generator/blob/main/examples/node/Dockerfile#L9)